src/syntax_highlight.rs

```diff
@@ -34,7 +36,8 @@ unsafe impl Sync for Setup {}
 lazy_static!{
     static ref SETUP: Setup = Setup {
         syntax_set: SyntaxSet::load_defaults_newlines(),
-        theme_set: ThemeSet::load_defaults()
+        theme_set: ThemeSet::load_defaults(),
+        theme_name: RefCell::new(DEFAULT_THEME_NAME.to_owned()),
```

Comment:
This feels a little dirty to me.

I'll need to look at this more to see if there is another reasonable feasible way.